### ISSUE TITLE
Cut off area and time values when period divisions /= 4

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/keplers-laws#176]
**Test device**
iPad
**Operating System**
iPadOS 17.0.1
**Browser**
Safari
**Problem description**
For https://github.com/phetsims/qa/issues/984
The Area and Time values are likely to be cut off by tool boxes or time controls even with the default orbit for any divisions other than 4.

**Visuals**
![IMG_0005](https://github.com/phetsims/keplers-laws/assets/41024075/b8b5f8c2-e40a-4657-83c5-1bf49389f666)
![IMG_0006](https://github.com/phetsims/keplers-laws/assets/41024075/4c72c38a-5716-4d8c-b707-53fe9484072b)
![IMG_0007](https://github.com/phetsims/keplers-laws/assets/41024075/5f5bffa8-d7c3-45d8-a934-78a2690d837e)
![IMG_0008](https://github.com/phetsims/keplers-laws/assets/41024075/0342060f-5d00-4267-b661-a4cd0a252e87)

**Comment by AgustinVallejo** [comment]
@DianaTavares @arouinfar, not really sure how to approach this one, as it's mostly a design issue, and I'm not sure how else we'd accomodate the number values.

**Comment by arouinfar** [comment]
@AgustinVallejo @DianaTavares I think some occlusion is inevitable and acceptable. However, I would expect the label for the default orbit to fit on screen without being occluded by panels. Can the default orbit be smaller? I can't remember if the current default has any significance. Or maybe the labels be moved closer to the areas? They seem to be pretty far from the areas they label (possibly to accommodate the velocity vector).

**Comment by DianaTavares** [comment]
I agree with Amy.  It is ok that with some orbits, the readouts are behind something. But it is not good that it is behind with the default. My suggestions,1) @AgustinVallejo to play with the position of that readouts, put them closer to the orbit. I think that the reason for them being where they are is for the velocity vector, but you can try to move them the most that is possible (especially play with the divisions where the label is to the right of the planet:

![image](https://github.com/phetsims/keplers-laws/assets/26031501/56619d75-f110-4174-ac58-81b614690e2c)

Probably this is nos something possible because the in the 6 division period, is not a way to move the readout somewhere closer:
![image](https://github.com/phetsims/keplers-laws/assets/26031501/2272bd38-d3b8-41e0-a6a5-90624c3a5300)

2) try moving the system more to the left.


3) finally, make the default orbit smaller. 

We need that in the default orbit, all the readouts of any period timer division are visible.

**Comment by AgustinVallejo** [comment]
I managed the following spacing by moving the center of the system:
![image](https://github.com/phetsims/keplers-laws/assets/37052186/a2a815be-9e0e-466b-a4ed-2509b83e158c)
![image](https://github.com/phetsims/keplers-laws/assets/37052186/582fed2d-0775-47fe-b8e8-c5e88b69d26d)

However I'm not super sold on the idea, as now orbis wouldn't be centered on the screen. See how a circular orbit would look now:
![image](https://github.com/phetsims/keplers-laws/assets/37052186/44486f0f-485e-4463-9076-81a5a630ed31)

I also tried changing the default zoom level but would have to zoom out too much... Not sure if I like changing the default orbit either, as there are other aesthetic considerations like it lining up with the grid lines. And it seems like the only orbit that begins lined with the grid lines and fits with this issue would literally be Earth's orbit (a=1, e=0). Other orbits fit but they are off grid. We could also change the grid. Althought it seems a really big design change, and not sure if it's ideal.

**Comment by AgustinVallejo** [comment]
Yesterday @DianaTavares @arouinfar and I met to discuss this, and we concluded that the side overlap might be unavoidable without introducing other design mishaps. However I'll try to reduce the vertical spacing so there's at least no overlap with the time controls.

**Comment by KatieWoe** [comment]
Behavior described in https://github.com/phetsims/keplers-laws/issues/176#issuecomment-1737824932 looks good in rc.2
